validator/delphi_validator/utils.py

```diff
@@ -13,6 +13,15 @@
     'national': r'^[a-zA-Z]{2}$'
 }
 
+
+def get_today():
+    """Wrapper function for mocking out date.today().
```

Comment:
may be able to use https://github.com/spulec/freezegun for tesiting

Reply:
Good idea, done